src/uwtools/drivers/ungrib.py

```diff
@@ -0,0 +1,175 @@
+"""
+A driver for the Ungrib model.
+"""
+
+import os
+import stat
+from datetime import datetime
+from pathlib import Path
+from typing import Any, Dict
+
+from iotaa import asset, dryrun, task, tasks
+
+from uwtools.config.formats.nml import NMLConfig
+from uwtools.drivers.driver import Driver
+from uwtools.strings import STR
+from uwtools.utils.tasks import file
+
+
+class Ungrib(Driver):
+    """
+    A driver for Ungrib.
```

Comment:
```suggestion
    A driver for ungrib.
```
We use an initial capital for the class name because that's Python convention, but a quick google for "wrf ungrib" indicates that the program/component itself is generally referred to in lower case.